config/phobos.yml.example

```diff
@@ -60,6 +60,11 @@ producer:
   # if greater than zero, the number of seconds between automatic message
   # deliveries. Only used for async_producer
   delivery_interval: 0
+  # Set this to true to cache the sync producer between publish calls.
+  # This can speed up subsequent messages by around 30%, but it does mean
+  # that you need to manually call sync_producer_shutdown before exiting,
+  # similar to async_producer_shutdown.
+  cache_sync_producer: false
```

Comment:
I suggest to rename this `persistent_connections` and default it to `false`. And if the connection is missing we can add some post install message that users look into this new feature and consider setting it to false (default) or migrate to true (requiring some minor changes). 

These minor changes requierd could described in a new README section:
```
1. [Installation](#installation)
  1. [Upgrade notes](#upgrading)
```

Reply:
Sounds great!